corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -326,6 +326,6 @@ $.fn.reportUserConfigurableConfigEditor = function (options) {
 $.fn.reportConfigList = function (options) {
     this.each(function(i, v) {
         var viewModel = new ReportConfigsViewModel(options);
-        ko.applyBindings(viewModel, $(this).get(i));
+        $(this).koApplyBindings(viewModel);
```

Comment:
Looked at this a bit more and I think this code might not be well-written. It doesn't make sense to use `i`, an index over the outer `this`, as an index on the inner `this`. If the original jQuery object matched multiple elements, you'd start with something like `[div0, div1]`, and then the inner loop will try to apply bindings to `[div0].get(0)` and then `[div1].get(1)`, which won't match anything.

Conveniently, this function is only ever called with an id selector, so the outer `this` never has more than one element and `i` is always zero. The same is true for all the similar pieces of code in https://github.com/dimagi/commcare-hq/commit/b0b003868c92637716f5c4c06652974f0e0985f3 - they're all only called on ids.

Thoughts?